### PR TITLE
issue 42, landing page

### PR DESCRIPTION
**Pull request by natapokie**
created landing page for orientation website ->

for testing, you should see this for desktop, 
image

and this for mobile, 
image

the instagram icon should link to the f!rosh instagram account

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+            {/*<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"/></svg> */}
+
+            {/* <svg src={InstagramLogo} className="instagram-svg-icon" alt="FontAwesome Instagram Logo" /> */}
```
Comment (Cali0707) [inline client/src/App.jsx:29]: Could you delete these comments?

Comment (Cali0707) [inline client/src/App.jsx:29]: Can you remove these comments?

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,43 +1,44 @@
 import { useState } from 'react';
-import logo from './logo.svg';
+// import InstagramLogo from './instagram-brands.svg';
```
Comment (Cali0707) [inline client/src/App.jsx:2]: Can you remove this unused import?

File: client/src/App.css [hunk 3]
```diff
@@ -1,42 +1,186 @@
...
+  /*background-attachment: fixed;*/
+
+  color: #ffffff;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+/* TITLE SECTION */
+.lp-title {
```
Comment (Cali0707) [inline client/src/App.css:29]: Can we rename this to just `title`?

File: client/src/App.css [hunk 4]
```diff
@@ -1,42 +1,186 @@
...
+  margin-bottom: 24px;
+}
+.subtitle {
+  font-size: 20px;
+  letter-spacing: 0.05em;
+  /*margin-bottom: 20px;*/
+}
+
+/* REGISTRATION INFO SECTION */
+.lp-info {
```
Comment (Cali0707) [inline client/src/App.css:66]: Can we rename this to just `info`?

File: client/src/App.css [hunk 5]
```diff
@@ -1,42 +1,186 @@
...
+  padding-left: 5%;
+  max-width: 751px;
+  text-align: left;
+  display: inline-block;
 }
 
-.App-link {
-  color: #61dafb;
+/* FOOTER */
+.lp-footer {
```
Comment (Cali0707) [inline client/src/App.css:95]: Can we rename this to just `footer`?

**Review by Cali0707 — CHANGES_REQUESTED** [review]
Overall looks amazing - great work! I left a few comments on specifics to change regarding commented-out code and the names of a few css classes, but otherwise this is great!

File: client/src/App.css [hunk 6]
```diff
@@ -1,42 +1,174 @@
...
+  top: 10px;
+  width: 851px;
+  height: 0px;
+  border: 2px solid #ffe76a;
+  border-radius: 100px;
+  margin-bottom: 24px;
+}
+.subtitle {
+  font-size: 20px;
+  letter-spacing: 0.05em;
```
Comment (jameskokoska) [inline client/src/App.css:56]: usually letter spacing isn't added unless it's really needed, it's okay to keep in though

File: client/src/App.jsx [hunk 7]
```diff
@@ -1,43 +1,38 @@
...
+          <hr className="line"></hr>
+          <p className="subtitle">
+            Hey F!rosh! We are working hard to give you the best experience, check back soon!
+          </p>
+        </div>
+
+        {/* Registration Info */}
+        <div className="info">
+          <a href="https://www.instagram.com/froshweek/" target="_blank" rel="noreferrer">
+            <img src={InstagramLogo} className="instagram-logo" />
```
Comment (jameskokoska) [inline client/src/App.jsx:23]: images should have an alt value, it's good practice (for accessibility). not needed though

**Review by jameskokoska — DISMISSED** [review]

**Comment by jameskokoska** [comment]
Looks good! Just some very minor things
They don't need to be changed, just a heads up
you can merge when your ready @natapokie

**Review by Cali0707 — APPROVED** [review]
This looks great, thanks for making those changes! I am merging this now.

**Pull request merged**